store/issue.go

```diff
@@ -538,9 +538,9 @@ func (*Store) findIssueImpl(ctx context.Context, tx *sql.Tx, find *api.IssueFind
 		args = append(args, *v)
 		args = append(args, *v)
 	}
-	if v := find.StatusList; v != nil {
+	if len(find.StatusList) != 0 {
```

Comment:
Do we still need this?

Reply:
Yeah, we cannot break the where clause: `where = append(where, fmt.Sprintf("status in (%s)", strings.Join(list, ",")))`